src/Microsoft.ML.ImageAnalytics/VectorToImageTransform.cs

```diff
@@ -340,7 +340,7 @@ protected override Delegate GetGetterCore(IChannel ch, IRow input, int iinfo, ou
             var ex = _exes[iinfo];
             bool needScale = ex.Offset != 0 || ex.Scale != 1;
             disposer = null;
-            var sourceType = Schema.GetColumnType(Infos[iinfo].Source);
+            var sourceType = OutputSchema.GetColumnType(Infos[iinfo].Source);
```

Comment:
>OutputSchema [](start = 29, length = 12)

This one is correct, but I think if you use InputSchema it would be more right thing to do.